Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py

```diff
@@ -111,26 +113,129 @@ def get_okta_fields(self):
 
         return okta_fields
 
+    def check_app_user_assignment(self, application_id, user_id):
+        uri = f'/apps/{application_id}/users/{user_id}'
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='response',
+            ok_codes=(200, 404)
+        )
+        return res.status_code == 200  # True if user is assigned to app, false otherwise
+
+    def list_apps(self, query, page, limit):
+        query_params = {
+            'q': encode_string_results(query),
+            'limit': encode_string_results(limit)
+        }
+
+        curr_page = 0
+        apps_batch, next_page = self.list_apps_batch(url_suffix='/apps', params=query_params)
```

Comment:
if you don't pass the page arg to the query, it means the API has its own page size? you do while till you reach the limit?